modin/experimental/pandas/io_exp.py

```diff
@@ -196,12 +196,12 @@ def _read(**kwargs) -> DataFrame:
     -------
     modin.DataFrame
     """
-    from modin.data_management.factories.dispatcher import EngineDispatcher
+    from modin.data_management.factories.dispatcher import FactoryDispatcher
```

Comment:
let's remove this `import`, it's redundant

Reply:
this import is [used](https://github.com/dchigarev/modin/blob/e4f8737cdb07a2de4ef59601987c4784093cec3b/modin/experimental/pandas/io_exp.py#L204) further in the code...

Reply:
but we `import` it globally [in the beginning](https://github.com/dchigarev/modin/blob/e4f8737cdb07a2de4ef59601987c4784093cec3b/modin/experimental/pandas/io_exp.py#L24) 😉